src/main/resources/db/migration/V1__create_table_users.sql

```diff
@@ -0,0 +1,14 @@
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+CREATE TABLE IF NOT EXISTS "users" (
```

Comment:
None of the `""` within this file should be necessary. Also, avoid giving tables plural names. This one can be named `user_account` to avoid the issue with using only `user`. Additionally, we can make migrations transactional by adding BEGIN; and COMMIT; to the start and end of the file (but not around creating extensions). That way, if a migration was to fail halfway through, everything that was done before would be rolled back and you wouldn't be left with some invalid half-state.